Cargo.toml

```diff
@@ -13,23 +13,21 @@ repository = "https://github.com/japaric/cortex-m-rtfm"
 version = "0.3.0"
 
 [dependencies]
-cortex-m = { git = "https://github.com/japaric/cortex-m" }
+cortex-m = "0.4.0"
+cortex-m-rtfm-macros = "0.3.0"
```

Comment:
Is it not possible to keep the `path` dependency? I *think* Cargo will automatically rewrite it and publish both crates together.

Reply:
I can try that next time.